### DISCUSSION TITLE
フォロー機能が動作しない

### DISCUSSION THREAD

**Discussion by nakamotojp** [increments/qiita-discussions#559] [ヘルプ]
すいません、次のQiitaで質問させていただいているのですが、フォロー機能が動作しません。

どうしたら動作しますでしょうか？

https://qiita.com/nakamoto/questions/4c04b60bedb431230799

**Comment by tanaka-asami415** [comment]
@nakamotojp 
ご報告ありがとうございます！質問を確認いたしました。
こちらでは症状が再現できなかっため、下記情報を教えていただけますと幸いです🙏

・デバイス、OS、ブラウザのバージョン

> フォロー中表示にならないし、自分のフォロー中の数も増えない

この時、数字をクリックして表示される「フォローしているユーザー」一覧にも
フォローボタンを押したユーザーさまが表示がされない状態でしょうか？

> ちなみに、Organizationをフォローしても自分がフォローしている数は増えない

Organizationのフォローは、[マイページ](https://help.qiita.com/ja/articles/qiita-mypage)の「フォロー中のOrganization」にアイコンが表示される仕様となっており、
数は、フォローしている（されている）ユーザー数の表示となります。
